compiler/src/iree/compiler/Codegen/Common/CPU/CPUMaterializeEncodingPass.cpp

```diff
@@ -206,6 +207,7 @@ enumerateMatmulTileArm64(TypeRange elementTypes, ExecutableTargetAttr target) {
 // are handled by transposition in chooseMatmulTile.
 static SmallVector<TileMxNxK>
 enumerateMatmulTileX86_64(TypeRange elementTypes, ExecutableTargetAttr target) {
+  return {};
```

Comment:
looks like a stray debugging helper :-)

Reply:
I should have double checked. no idea why it was included here ... removed.